src/main/java/org/folio/circulation/domain/NoteBuilder.java

```diff
@@ -0,0 +1,41 @@
+package org.folio.circulation.domain;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class NoteBuilder {
```

Comment:
@bohdan-suprun Could much of this be avoided by using some lombok annotations?

Reply:
Yes, you can place `@Builder` annotation for the `Note` class (it also generates all args constructor). You can also annotate the `links` field with `@Singular` annotation it will add a `link(NoteLink)` method that allows to add one element to the array.

You can also add `@Getter` annotation to generate getters for your fields. 

Here is the class with all annotations:
```java
@Builder
@Getter
public class Note {
  private final String id;
  private final String typeId;
  private final String domain;
  private final String title;
  private final String content;
  @Singular
  private final List<NoteLink> links;

  public static Note from(JsonObject jsonObject) {...}

  public JsonObject toJson() {...}
}
```